docs/pages/guides/local-first.mdx

```diff
@@ -64,6 +64,11 @@ It works with Expo and React Native (via [React Native Async Storage](https://gi
 
 [Prisma](https://prisma.io) is well known as the most popular ORM for Node.js and TypeScript backends, and it's now available for [Expo and React Native in early access](https://www.prisma.io/blog/bringing-prisma-orm-to-react-native-and-expo). Prisma aims to provide a complete local-first solution, with state management, syncing, and persistence all covered for you. While it's still early, [Beto Moedano](https://github.com/betomoedano) has put together full walkthrough of using Prisma with Expo to build a local-first Notion clone, [watch it on YouTube](https://www.youtube.com/watch?v=uTrPte0sCiw) and [check out the code on GitHub](https://github.com/betomoedano/React-Native-Notion-Clone).
 
+### Jazz
+
+[Jazz.tools](https://jazz.tools/docs/react-native) is a framework for building local-first apps. It is open source, and you can self-host it or use [Jazz Cloud](https://jazz.tools/cloud) to start quickly. [Jazz](https://jazz.tools) provides first-class support for Expo.
+To learn more, check out the [examples repository](https://github.com/garden-co/jazz/tree/main/examples) or see the [Getting Started Guide](https://jazz.tools/docs/react-native) for detailed instructions.
```

Comment:
It might be friendlier to link to https://jazz.tools/examples - we'll set up an anchor so you can link straight to React Native examples